status/pcpp/.gitignore

```diff
@@ -0,0 +1 @@
+*.pyc
```

Comment:
You may want to add `*.pyo` and `__pycache__` too

Reply:
Done